src/build_tools/hooks_hydra.py

```diff
@@ -381,9 +381,11 @@ def end_hook():
     fancylogger.setLogLevelInfo()
 
     # submit Lmod cache job
-    if os.getenv('BUILD_TOOLS_RUN_LMOD_CACHE'):
+    if os.getenv('BUILD_TOOLS_RUN_LMOD_CACHE', '1'):
```

Comment:
are you sure this works as intended? Doesn't `export BUILD_TOOLS_RUN_LMOD_CACHE=` return true here?

Reply:
> are you sure this works as intended? Doesn't `export BUILD_TOOLS_RUN_LMOD_CACHE=` return true here?

no, an empty string is not `None`, but still evaluates to `False`.

but i am not against doing a hard `== 1` if you prefer

Reply:
I would be inclined to go for a hard `== 1` to have clear behaviour. Only a 1 is sets it, everything else is `False`.